### ISSUE TITLE
Can I use my own voice？ 可以用自己的声音吗

### ISSUE THREAD

**Issue by zhuya1996** [JiehangXie/PaddleBoBo#34]
Can I use my own voice？ 可以用自己的声音吗

**Comment by JiehangXie** [comment]
Yes, you can use your own sound wavs files to replace the voice TTS moduled generated. If you want to automatically generate your imitation sound, you may consider using voice clone model in PaddleSpeech project.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "JiehangXie/PaddleBoBo"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年6月23日(星期四) 下午5:33
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[JiehangXie/PaddleBoBo] Can I use my own voice？ 可以用自己的声音吗 (Issue #34)





 
Can I use my own voice？ 可以用自己的声音吗

**Comment by zhuya1996** [comment]
您好，Can you give me a tutorial

**Comment by JiehangXie** [comment]
PaddleSpeech/examples/aishell3/vc1 at develop · PaddlePaddle/PaddleSpeech (github.com)




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "JiehangXie/PaddleBoBo"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年6月23日(星期四) 晚上6:03
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [JiehangXie/PaddleBoBo] Can I use my own voice？ 可以用自己的声音吗 (Issue #34)





 
您好，Can you give me a tutorial
